functions/go/apply-setters/applysetters/apply_setters_test.go

```diff
@@ -366,36 +366,25 @@ spec:
         - name: nginx
           image: nginx:1.7.9 # kpt-set: ${image}:${tag}
 `,
-			errMsg: `failed to configure function: input setters list cannot be empty`,
+			errMsg: `input setters list cannot be empty`,
 		},
 		{
 			name: "set empty values",
 			input: `apiVersion: v1
 kind: Service
 metadata:
   name: myService # kpt-set: ${app}
-  namespace: foo # kpt-set: ${ns}
-image: nginx:1.7.1 # kpt-set: ${image}:${tag}
-env: # kpt-set: ${env}
-  - foo
-  - bar
 `,
 			config: `
 data:
   app: ""
```

Comment:
This should be allowed. `null` values are not allowed, but empty strings are.

Reply:
Nvm. Empty strings and null are also allowed. Not specifying value is not allowed. Closing this PR.